docs/operations/blocks-storage.md

```diff
@@ -123,6 +123,14 @@ tsdb:
   # CLI flag: -experimental.tsdb.ship-concurrency
   [ship_concurrency: <int> | default = 10]
 
+  # How frequently does Cortex try to compact TSDB head. Block is only created if data covers smallest block range.
+  # CLI flag: -experimental.tsdb.head-compaction-interval
+  [head_compaction_interval: <duration> | default = 2h]
```

Comment:
Hrm, this should be `5m` and the tests should fail. Can you check why its not failing?

Reply:
I have edited this file manually. Somehow it isn't covered by Marco's tool, but haven't looked into more detail why yet. I'll fix the default.

Reply:
I think the reason is simply because block-storage is a separate file, while doc tool only generates config-file-reference.md. @pracucci surely knows better, I'm just guessing based on 5min looking around.